src/main/resources/application.yaml

```diff
@@ -78,4 +79,20 @@ identity-account-mapper:
     register-beneficiary: /beneficiary
     add-payment-modality: /paymentModality
     update-payment-modality: /paymentModality
-    account-lookup: /beneficiary
\ No newline at end of file
+    account-lookup: /beneficiary
+
+kafka:
+  brokers: "kafka:9092"
+  topic: "zeebe-export"
+  consumerTimeoutMs: 6000
+
+zeebe-test:
+  enabled: true
+
+zeebe-operations:
+  contactpoint: "https://zeebeops2.sandbox.fynarfin.io"
```

Comment:
this doesn't look like a valid URL(zeebeops2 is not available in the HELM charts)
also, these changes to be added to the integration test Job in the env template as service names so that this test can run when running as a Job.